spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -347,8 +347,10 @@ def main(argv: Sequence[str]):
             _, ext = os.path.splitext(arguments.config)
             if ext == '.json':
                 config = json.load(conf)
-            if ext == '.yml' or ext == '.yaml':
+            elif ext == '.yml' or ext == '.yaml':
                 config = yaml.load(conf, Loader=yaml.Loader)
+            else:
+                raise ValueError('Unrecognized configuration file type: {}'.format(ext))
```

Comment:
I was going to say that CLI scripts usually use `printv` and `sys.exit` for error messages, but it looks like `sct_run_batch` consistently uses `print` and `raise` instead, so this is fine.